cfme/configure/access_control.py

```diff
@@ -1195,7 +1182,52 @@ def set_role_product_features(self, view, product_features):
         return feature_update
 
 
-@navigator.register(Role, 'All')
+@attr.s
+class RoleCollection(BaseCollection):
+    ENTITY = Role
+
+    def create(self, name=None, vm_restriction=None, product_features=None, cancel=False):
+        """ Create role method
+            Args:
+                cancel: True - if you want to cancel role creation,
+                        by default, role will be created
```

Comment:
**REQUIRED**: Dedent this block and put a newline before Args and Throws. Also Throws should be called Raises.

Reply:
Fixed - was a copy pasta :)